DenunciaCiudadana/src/actions/placesActions.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import firestore from '@react-native-firebase/firestore';
+import {types} from '../types/type';
+
+export const getPlacesAPI = value => {
+  const API_URL = 'https://api.copomex.com/query';
+  const API_METHOD = 'info_cp';
+  const API_SEARCH = value;
+  const API_PARAMS = 'type=simplified';
```

Comment:
I recommend to put all these constant in another file with name "service" or something like that to be more organized your files.